### PR TITLE
Add PackageReleaseNotes to MSAL

### PR DESCRIPTION
**Pull request by bgavrilMS**
#1461

### CODE REVIEW

File: src/client/Directory.Build.props [hunk 1]
```diff
@@ -35,6 +35,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageReleaseNotes>A changelog is available at https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/changelog.txt </PackageReleaseNotes>
```
Comment (jmprieur) [inline src/client/Directory.Build.props]: Thanks @bgavrilMS this is an awesome improvement.
I wonder if we should not add 
```
and the roadmap (including blog post for releases) from https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap
```

Comment (bgavrilMS) [reply src/client/Directory.Build.props]: Ok, but let's exclude the parenthesis - too many words for such a small place.

File: src/client/Directory.Build.props [hunk 2]
```diff
@@ -35,6 +35,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageReleaseNotes>The changelog is available at https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/changelog.txt and the roadmap at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap </PackageReleaseNotes>
```
Comment (jennyf19) [inline src/client/Directory.Build.props]: this is pointing to adal's release notes...am i missing something? @jmprieur @bgavrilMS

Comment (jmprieur) [reply src/client/Directory.Build.props]: Good catch, Jenny. My bad

Comment (jennyf19) [reply src/client/Directory.Build.props]: @jmprieur i fixed it.

Comment (jmprieur) [reply src/client/Directory.Build.props]: @jennyf19 thank you.

**Pull request merged**